src/main.rs

```diff
@@ -1,3 +1,33 @@
 fn main() {
-    println!("Hello, world!");
+    println!("Olá! 😃");
+
+    loop {
+        println!();
+        println!("Você gostaria de adicionar um novo TODO? 🤔 (Digite: 's' para SIM ou qualquer outra tecla para NÃO)");
```

Comment:
Coisinha minor, você não precisa usar dois `println!`s aqui:
```suggestion
        println!("\nVocê gostaria de adicionar um novo TODO? 🤔 (Digite: 's' para SIM ou qualquer outra tecla para NÃO)");
```

Mesma coisa nos outros casos.

Reply:
Top tirar D+ tirar mais um ```printfn!``` !! Valeu

Reply:
Mias alguns na verdade!! :D